hi-ml-azure/testazure/testazure/test_himl.py

```diff
@@ -1509,3 +1509,24 @@ def test_extract_v2_inputs_outputs_from_args() -> None:
         input_datasets, output_datasets = himl._extract_v2_inputs_outputs_from_args()
         assert len(input_datasets) == 0
         assert len(output_datasets) == 0
+
```

Comment:
It would be good to add a test that all the expected arguments of `command` are passed in

Reply:
Added